src/store/whiteboard.ts

```diff
@@ -1,10 +1,35 @@
 import _ from 'lodash'
 import {createWhiteboardElement, getWhiteboard, updateWhiteboardElement} from '@/api/whiteboards'
 
-export function $_findElement(state: any, uid: number) {
-  return _.find(state.board.elements, ['uid', uid])
+const $_createWhiteboardElement = (commit: any, elementType: string, whiteboardId: number) => {
+  const element = {
+    canvas: {
+      backgroundColor: 'green',
+      height: window.innerHeight - 50,
+      type: 'canvas',
+      width: window.innerWidth,
+    },
+    ellipsis: {
+      fill: 'rgb(0,0,0)',
+      type: 'ellipsis'
+    },
+    text: {
+      fill: 'rgb(0,0,0)',
+      text: '',
+      type: 'text'
+    }
+  }[elementType]
+  return createWhiteboardElement(element, whiteboardId).then(data => {
+    const element = _.get(data, 'element')
+    console.log(`element: ${JSON.stringify(element)}`)
```

Comment:
@johncrossman Is this log statement meant to be here?

Reply:
Good catch. I'll remove.